trestlebot/transformers/cac_transformer.py

```diff
@@ -294,26 +293,11 @@ def _load_rule_yaml(self, rule_obj: RuleInfo) -> None:
         Args:
             rule_obj: The rule object where collection rule data is stored.
         """
-        # The function will raise error: FileNotFoundError
-        # which is caused by loading jinja macros, and this
-        # needs to be fixed by ssg. So comment blow temporarily.
-        # rule_file = ssg.rules.get_rule_dir_yaml(rule_obj.rule_dir)
-        # rule_yaml = ssg.build_yaml.Rule.from_yaml(rule_file)
-        # rule_yaml.normalize(self.product)
-        # description = self._clean_rule_description(rule_yaml.description)
-        # rule_obj.add_description(description)
-        rule_obj.add_description("Pending update")
+        rule_file = get_rule_dir_yaml(rule_obj.rule_dir)
+        rule_yaml = open_and_macro_expand_from_dir(rule_file, self.root)
```

Comment:
To properly render the Jinja2 macros we should load the product yaml (load_product_yaml) and pass it as initial `substitutions_dict` for `open_and_macro_expand_from_dir`. This should solve the issue with test units.